packages/mdc-theme/_mixins.scss

```diff
@@ -20,8 +20,47 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "./variables";
 
+@mixin mdc-theme($query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  :root {
+    @include mdc-feature-targets($feat-color) {
```

Comment:
Is there a reason this feature-targets is outside the `@each` but the ones below are inside?

Reply:
This one doesn't have any selectors inside of it, just properties